modules/nf-commons/src/main/nextflow/plugin/PluginUpdater.groovy

```diff
@@ -221,13 +221,20 @@ class PluginUpdater extends UpdateManager {
         // 2. download to temporary location
         Path downloaded = safeDownloadPlugin(id, version);
 
-        // 3. unzip the content and delete downloaded file
+        // 3. rename if filename is sha digest
```

Comment:
I'd try to provide a bit more context on the rationale and an example of the URI to make it more directly understandable

Reply:
Did it